samples/hello-world-js/src/MainPanel.js

```diff
@@ -73,28 +73,30 @@ export default class MainPanel extends RX.Component{
 
     render() {
         return (
-            <RX.ScrollView style={ styles.scroll }>
-                <RX.View style={ styles.container }>
-                    <RX.Animated.Text style={ [styles.helloWorld, this._animatedStyle] }>
-                        Hello World
-                    </RX.Animated.Text>
-                    <RX.Text style={ styles.welcome }>
-                        Welcome to ReactXP
-                    </RX.Text>
-                    <RX.Text style={ styles.instructions }>
-                        Edit App.js to get started
-                    </RX.Text>
-                    <RX.Link style={ styles.docLink } url={ 'https://microsoft.github.io/reactxp/docs' }>
-                        View ReactXP documentation
-                    </RX.Link>
-
-                    <RX.Button style={ styles.roundButton } onPress={ this.props.onPressNavigate }>
-                        <RX.Text style={ styles.buttonText }>
-                            See More Examples
+            <RX.View useSafeInsets={true}>
```

Comment:
Won't this require a style with flex:1 and alignSelf: 'stretch'? Same with other RX.View containers you added to other samples.

Reply:
Since `flex: 1` is always recommended/required, I've added an override in the View class itself to always apply that style.  I didn't think to also apply alignSelf: 'stretch', should I?  Which dev-error would that overcome?

Reply:
Also, do you think it's worth adding a note to the doc that those styles are auto-applied?

Reply:
Yeah, you should add alignSelf: 'stretch' as well. Otherwise the layout engine, by default, sizes the non-axis direction according to the parent's alignItems attribute and the width/height of the children. 

And yeah, the documentation should reflect this style override. Also, the doc should probably also indicate that other props may be ignored when this prop is true.